src/obiwan/core.clj

```diff
@@ -227,6 +228,16 @@
 (defn zrange [^UnifiedJedis redis k zmin zmax]
   (.zrange redis k zmin zmax))
 
+(defn <-tuple
+  "this fn transforms the data from redis tuple to clojure map"
+  [^Tuple v]
+  {(.getScore v) (.getElement v)})
+
```

Comment:
let's move this to [obiwan.commands](https://github.com/tolitius/obiwan/blob/master/src/obiwan/commands.clj)